app/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,13 @@
+arguments=
```

Comment:
I am not sure if this file needs to be part of the repo or can be added to .gitignore.

Reply:
The file was generated by a vscode extension but invisible in my editor, so I did not even notice it. I have added it to gitignore